ibm/service/cos/resource_ibm_cos_replication_configuration.go

```diff
@@ -0,0 +1,438 @@
+package cos
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	bxsession "github.com/IBM-Cloud/bluemix-go/session"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/ibm-cos-sdk-go/aws"
+	"github.com/IBM/ibm-cos-sdk-go/aws/credentials/ibmiam"
+	token "github.com/IBM/ibm-cos-sdk-go/aws/credentials/ibmiam/token"
+	"github.com/IBM/ibm-cos-sdk-go/aws/session"
+	"github.com/IBM/ibm-cos-sdk-go/service/s3"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	validation "github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceIBMCOSBucketReplicationConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceBucketReplicationConfigurationCreate,
```

Comment:
Why the name of this method is different from others (resourceBucket...) and others (resourceIBMCOSBucket...)